lib/data/monorepo.json

```
@@ -56,10 +56,26 @@
     ],
     "awsappsync": "https://github.com/awslabs/aws-mobile-appsync-sdk-js",
     "axum": "https://github.com/tokio-rs/axum",
+    "azure-azure-libraries-for-net": "https://github.com/Azure/azure-libraries-for-net",
+    "azure-azure-sdk-for-android": "https://github.com/Azure/azure-sdk-for-android",
+    "azure-azure-sdk-for-c": "https://github.com/Azure/azure-sdk-for-c",
+    "azure-azure-sdk-for-cpp": "https://github.com/Azure/azure-sdk-for-cpp",
+    "azure-azure-sdk-for-go": "https://github.com/Azure/azure-sdk-for-go",
+    "azure-azure-sdk-for-ios": "https://github.com/Azure/azure-sdk-for-ios",
+    "azure-azure-sdk-for-java": "https://github.com/Azure/azure-sdk-for-java",
+    "azure-azure-sdk-for-js": "https://github.com/Azure/azure-sdk-for-js",
+    "azure-azure-sdk-for-net": "https://github.com/Azure/azure-sdk-for-net",
+    "azure-azure-sdk-for-python": "https://github.com/Azure/azure-sdk-for-python",
+    "azure-azure-sdk-for-rust": "https://github.com/Azure/azure-sdk-for-rust",
+    "azure-azure-storage-net": "https://github.com/Azure/azure-storage-net",
     "azure-functions-dotnet-worker": "https://github.com/Azure/azure-functions-dotnet-worker",
```

If we want I could also change this

```suggestion
    "azure-azure-functions-dotnet-worker": "https://github.com/Azure/azure-functions-dotnet-worker",
```

---

Do you think we really need the double azure-azure in the new ones? Is there any confusion or risk if a single azure- is used?

---

Taking a sample: `https://github.com/Azure/azure-sdk-for-rust`

I was thinking `${org}.${repoName}` so first I tried `Azure.azure-sdk-for-rust` but `.` were not allowed, as well as everything has to be lower case. So I had to replace `Azure` with `azure` and `.` with `-` that's how it was conceived.

In theory another org `MuchBetterAzure` can fork `azure-sdk-for-rust` and then how do we keep them apart?

Not likely this would happen but naming collisions might happen especially for more generically named repos such as `autocomplete` which today looks like this:

```
"algoliasearch-autocomplete": "https://github.com/algolia/autocomplete",
```

---

We don't have a fixed rule. We need the name to be long enough that it won't be confused for other, similarly named repos

---

Got it, removed unnecessary double `azure-` ✅  but still kept the prefix so they are appearing together

---

@rarkins fixed comments